cmd/test-utils_test.go

```diff
@@ -366,6 +367,20 @@ func StartTestStorageRPCServer(t TestErrHandler, instanceType string, diskN int)
 	testRPCServer.AccessKey = credentials.AccessKey
 	testRPCServer.SecretKey = credentials.SecretKey
 
+	// Create dummy format json file.
+	emptyConfig := []byte("{}")
```

Comment:
We need to keep it similar to what would be expected in a good condition, a dummy has no real meaning here and no one can build context by looking at these lines.